pkg/db/models/prow.go

```diff
@@ -132,8 +133,8 @@ type Bug struct {
 	FixVersions     pq.StringArray `json:"fix_versions" gorm:"type:text[]"`
 	Components      pq.StringArray `json:"components" gorm:"type:text[]"`
 	URL             string         `json:"url"`
-	Tests           []Test         `json:"-" gorm:"many2many:bug_tests;"`
-	Jobs            []ProwJob      `json:"-" gorm:"many2many:bug_jobs;"`
+	Tests           []Test         `json:"-" gorm:"many2many:bug_tests;constraint:OnDelete:CASCADE;"`
+	Jobs            []ProwJob      `json:"-" gorm:"many2many:bug_jobs;constraint:OnDelete:CASCADE;"`
```

Comment:
Does this *only* affect the bug_tests/bug_jobs entries or will it flow to actual bugs and jobs? If it does we've got an issue, stale bugs from search.ci that are no longer in the results there but are in our db will get cleaned out, and right now I think that happens often if the bug hasn't been touched in 3 weeks. (we need to talk to Hoxna about that actually) Disappearing from the query would be fairly common.

Reply:
Perfect thanks.